Source/MediaStorageAndFileFormat/gdcmFileStreamer.cxx

```diff
@@ -22,11 +22,11 @@
 #include "gdcmEvent.h"
 #include "gdcmProgressEvent.h"
 
-#define _FILE_OFFSET_BITS 64
+#define FILE_OFFSET_BITS 64
```

Comment:
It's correct in fact, see:  https://clang.llvm.org/extra/clang-tidy/checks/bugprone/reserved-identifier.html

Basically: starting with underscore is reserved.  And since this is in a .cxx, not .h, it seems safe to rename.